subjects/betweenus/README.md

```diff
@@ -0,0 +1,38 @@
+## between-us
+
+write a function named `BetweenUs` that takes 3 paramters and return : 
+- if the first paramter is between the second and third paramters, return **true** else return **false**
+
+### Function
+```go
+func BetweenUs(num, min, max int) bool {
+    // Your code here
+}
+```
+
+
+### Usage
+```go
+package main
+
+import "fmt"
+
+func main(){
+    fmt.Println(BetweenUs(1, 2, 3))
+    fmt.Println(BetweenUs(1, 1, 3))
+    fmt.Println(BetweenUs(1, 3, 3))
+    fmt.Println(BetweenUs(1, 1, 1))
+    fmt.Println(BetweenUs(1, 2, 1))
```

Comment:
Should this be false? Probably it should say that if the min>max the function must return false.